app/front/scripts/services/data-package-api/index.js

```diff
@@ -15,31 +16,50 @@ function loadConfig(settingsUrl) {
     module.exports.apiConfig = config.api;
     module.exports.searchConfig = config.search;
     module.exports.dataMineConfig = config.dataMine;
+    module.exports.osExplorerUrl = config.osExplorerUrl;
     return config;
   });
 }
 
-function getDataPackages(authToken) {
+function getDataPackages(authToken, packageId, userId) {
   return loadConfig().then(function() {
     var searchConfig = module.exports.searchConfig;
     var url = searchConfig.url + '?size=10000';
     if (authToken) {
       url += '&jwt=' + encodeURIComponent(authToken);
     }
+    if (packageId) {
```

Comment:
Search API should return correct responses now.
Having both `id` and `package.owner` in the URL is meaningless, as these conditions are `AND`ed and not `OR`ed.